nginx/service.sls

```diff
@@ -16,7 +19,10 @@ include:
 nginx_systemd_service_file:
   file.managed:
     - name: /lib/systemd/system/nginx.service
-    - source: salt://nginx/files/nginx.service
+    - source: {{ files_switch(['nginx.service'],
+                              'nginx_systemd_service_file_file_managed'
```

Comment:
The state name is fine here (and I've given that in the `pillar.example` comment as well):

```suggestion
                              'nginx_systemd_service_file'
```